lib/logger.js

```diff
@@ -41,8 +61,22 @@ const streamConverter = {
         };
     },
     stdout(stream, conf) {
+        if ( stream.named_levels ) {
+            return {
+                type: 'raw',
+                stream: new NamedLevelStdout(process.stdout),
+                level: stream.level || conf.level
+            };
+        } else {
+            return {
+                stream: process.stdout,
+                level: stream.level || conf.level
+            };
+        }
+    },
+    namedstdout(stream, conf) {
```

Comment:
I guess we no longer need this function?

Reply:
Of course